tests/conftest.py

```diff
@@ -80,3 +83,70 @@ def sqlite_engine():
 @pytest.fixture
 def Calendar():
     return calendar.Calendar(0)
+
+
+@pytest.fixture
+def no_event_user(session):
+    """a user made for testing who doesn't own any event."""
+    user = create_user(
+        session=session,
+        username='new_test_username',
+        password='new_test_password',
+        email='new2_test.email@gmail.com',
+        language_id='english'
+    )
+
+    return user
+
+
+@pytest.fixture
+def event_owning_user(session):
+    """a user made for testing who already owns an event."""
+    user = create_user(
+        session=session,
+        username='new_test_username2',
+        password='new_test_password2',
+        email='new_test_love231.email@gmail.com',
+        language_id='english'
+    )
+
+    data = {
+        'title': 'event_owning_user event',
+        'start': datetime.strptime('2021-05-05 14:59', '%Y-%m-%d %H:%M'),
+        'end': datetime.strptime('2021-05-05 15:01', '%Y-%m-%d %H:%M'),
+        'location': 'https://us02web.zoom.us/j/875384596',
+        'content': 'content',
+        'owner_id': user.id,
+    }
+
+    create_event(session, **data)
+
+    return user
+
+
+@pytest.fixture
+def user1(session):
+    """another user made for testing"""
+    user = create_user(
+        session=session,
+        username='user2user2',
+        password='verynicepass',
+        email='trulyyours1.email@gmail.com',
+        language_id='english'
+    )
+    return user
```

Comment:
Don't create a new user fixture, use one of the others.

Reply:
i have to use several different users since i'm testing mailing list.